packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -280,6 +281,20 @@ class RawAutocomplete<T extends Object> extends StatefulWidget {
   /// This parameter is ignored if [textEditingController] is defined.
   final TextEditingValue? initialValue;
 
+  /// {@template flutter.widgets.RawAutocomplete.optionsLayerLink}
+  /// Links the [optionsBuilder] to a custom [CompositedTransformTarget],
+  /// enabling control over the positioning and size of the options overlay.
+  ///
+  /// A [LayerLink] is required to change the location or size of the [optionsBuilder].
+  /// When setting [optionsLayerLink], [RawAutocomplete] must have a [CompositedTransformTarget]
+  /// as a parent widget to link the options overlay to the correct position in the widget tree.
+  ///
+  /// If a [LayerLink] is not provided or null, [RawAutocomplete] uses the default size,
+  /// and [LayerLink] is created internally,
+  /// with a [CompositedTransformTarget] widget added to the widget tree by default.
```

Comment:
```suggestion
  /// If null, this widget will create its own [LayerLink] and add a [CompositedTransformTarget] to the widget tree.
```
A bit more concise, and removed information that can't already be inferred from another segment of the documentation.